samples/bluetooth/mesh/light/Kconfig

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Bluetooth mesh light sample"
+
+config BT_MESH_LIGHT_SAMPLE_NRF52_DFU
+	bool "Enable SMP DFU support"
+	depends on MCUMGR_CMD_OS_MGMT
+	depends on MCUMGR_CMD_IMG_MGMT
+	depends on MCUMGR_SMP_BT
```

Comment:
Aren't these options enabled in overlay file?

Reply:
Yes, they are. But `depends on` does not pick the option as far as I know, it only makes the compiler deselect the dependent option unless all `depends on` requirements are fulfilled. I choose those dependencies since they are required to build `smp_dfu.c`

Reply:
But aren't they enabled if use overlay file? How do you see a user should enable dfu: through overlay file or through menuconfig, or somehow else?

Reply:
I guess it is just redundancy. I want the user to use the overlay to enable dfu

Reply:
I can remove it